notebooks/02.08-Sorting.ipynb

```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>Up to this point</blockquote><p>alright!</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>